opendevin/llm/llm.py

```diff
@@ -303,9 +304,11 @@ def completion_cost(self, response):
                 )
                 self.metrics.add_cost(cost)
                 return cost
-            except Exception:
+            except Exception as e:
                 self.cost_metric_supported = False
-                logger.warning('Cost calculation not supported for this model.')
+                logger.warning(
```

Comment:
Please no "e" here. This is a one-time logging, that is expected if litellm reports it back as unsupported.